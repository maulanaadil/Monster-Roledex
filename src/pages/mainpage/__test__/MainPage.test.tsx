import React from 'react';
import { shallow } from 'enzyme';
import MainPage from '../index';

describe('renders MainPage page', () => {
  let wrapper: any;

  beforeEach(() => {
    wrapper = shallow(<MainPage />);
  });

  it('should render same snapshots of MainPage page', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
