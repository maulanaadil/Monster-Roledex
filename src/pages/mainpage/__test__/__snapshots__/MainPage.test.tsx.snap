// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders MainPage page should render same snapshots of MainPage page 1`] = `
<styled.div>
  <styled.h1>
    Monster Roledex
  </styled.h1>
  <SearchBox
    handleChange={[Function]}
    placeholder="Search Monster"
  />
  <CardList
    monster={Array []}
  />
</styled.div>
`;
